circuits/kimchi/src/gates/generic.rs

```diff
@@ -27,27 +27,19 @@ impl<F: FftField> CircuitGate<F> {
     }
 
     /// creates an addition gate
-    pub fn create_generic_add(row: usize, wires: GateWires) -> Self {
```

Comment:
I was missing the fact that we can do `left_coeff * w0 + right_coeff * w1`, not just `w0 + w1`